doc/whats_new.rst

```diff
@@ -55,6 +57,8 @@ API
 
 - Replace parameter `T` with `tstop` in :func:`~hnn_core.Network.add_tonic_bias` and :func:`~hnn_core.Cell.create_tonic_bias` to be more consistent with other functions and improve readability, by `Kenneth Loi`_ in `#354 <https://github.com/jonescompneurolab/hnn-core/pull/354>`_
 
+- Previously published models can now be loaded via ``net=law_model()`` and ``jones_2009_model()``, by `Nick Tolley`_ in `#348 <https://github.com/jonescompneurolab/hnn-core/pull/348>`_
```

Comment:
goes into `Changelog` section. Only API "changes" go into API section. It's to inform people what they need to update in their code.